games-roguelike/FTL-gog/FTL-gog-1.6.7.ebuild

```diff
@@ -47,7 +47,14 @@ src_prepare() {
 src_install() {
 	insinto /opt/gog/FTL
 	doins -r .
-	fperms 755 /opt/gog/FTL/{start.sh,game/{FTL,data/FTL*}}
+	fperms 755 /opt/gog/FTL/{start.sh,game/FTL}
```

Comment:
`+x`?

Reply:
I understand the difference between what the octal notation dictates and what `+x` dictates, but in this context the end result is the same. Why change this? `qgrep` shows there are plenty of instances of both notations across the tree.

Reply:
The reason is that you don't use `644` explicitly above, so it's cleaner not to assume it. Even if PMS guarantees it.

Reply:
Certainly does make sense, thanks. Switched to `+x`